app/angulartics.matomo/index.js

```diff
@@ -0,0 +1,192 @@
+import angular from 'angular';
+// TODO move to build env-var?
```

Comment:
Chaim is it a TODO or a suggestion, should we do it directly or add a ticket for followup?